packages/ember-glimmer/tests/integration/components/closure-components-test.js

```diff
@@ -40,6 +40,142 @@ moduleFor('@htmlbars Components test: closure components', class extends Renderi
     this.assertText('Hodi Hodari');
   }
 
+  ['@test keeps nested rest positional parameters if rendered with no positional parameters']() {
```

Comment:
I add that to the two that made sense to me. Let me know if you want it added to the rest!